platform-includes/replay/privacy-configuration/javascript.mdx

```diff
@@ -3,6 +3,7 @@ you can opt out of the default text masking and image blocking by configuring th
 
 ```javascript
 Sentry.replayIntegration({
+  // NOTE: This will disable built-in masking. Look at other options opt-in to masking sensitive data
```

Comment:
Not sure why the second sentence is supposed to mean

Reply:
I would say, set these options as 'true' to mask sensitive data

Reply:
If we really want to add a comment here, I think first sentence will suffice